src/main/java/gov/usds/case_issues/model/CaseSnoozeFilter.java

```diff
@@ -4,12 +4,12 @@
  * Categories of filter that a user of the general case list API might request.
  */
 public enum CaseSnoozeFilter {
-	/** All cases that are not currently snoozed: the original "active" list */
-	ACTIVE,
+	/** All cases */
+	ALL,
```

Comment:
This seems inconsistent with the previous plan to make the filter take a set of snooze states rather than a single value. I think rather than enumerating all the valid combinations, we should just make the back end actually honor a multi-valued parameter.

Reply:
And I think there are now four states, right? Untouched, snoozed, snooze-expired, snooze-terminated?